### PR TITLE
modules: fix examples to use FQCN

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
modules: fix examples to use FQCN

##### ISSUE TYPE
- Docs Pull Request

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

**Comment by Andersson007** [comment]
https://github.com/ansible-collections/overview/issues/45#issuecomment-648405709

**Comment by Andersson007** [comment]
fixed.
ready_for_review

**Comment by Andersson007** [comment]
CC @abadger @acozine

File: plugins/modules/identity/ipa/ipa_user.py [hunk 1]
```diff
@@ -117,26 +117,26 @@
     gidnumber: '100'
     homedirectory: /home/pinky
     ipa_host: ipa.example.com
-    ipa_user: admin
+    community.general.ipa_user: admin
```
Comment (felixfontein) [inline plugins/modules/identity/ipa/ipa_user.py:120]:
```suggestion
    ipa_user: admin
```
Happens three more times in this module.

Comment (Andersson007) [reply plugins/modules/identity/ipa/ipa_user.py]: @felixfontein thanks for catching this!
fixed

**Comment by felixfontein** [comment]
@Andersson007 thanks a lot for fixing this!

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing and merging!

**Pull request merged**